Header header
float32[] stateMean
float32[] stateCovariance
float32[] seenLandmarks
